__tests__/app.test.js

```diff
@@ -35,4 +35,86 @@ describe('3. get /api/topics', () => {
             })
         })
     });
+});
+
+describe('4. get /api/articles/:article_id + 7. comment_count', () => {
+    test('should return 200 and an object that matches the article_id for selected columns', () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toEqual({
+                    author: "butter_bridge",
+                    title: "Living in the shadow of a great man",
+                    article_id: 1,
+                    body: "I find this existence challenging",
+                    topic: "mitch",
+                    created_at: "2020-07-09T20:11:00.000Z",
+                    votes: 100,
+                    comment_count: 11
```

Comment:
Using objectContaining or hasProperty rather than "toEqual" could have meant that you didn't need to go back and change a previous test, and made the test a bit more flexible for future functionality. This test works for the current functionality though 👍